bitsail-connectors/connector-local-csv/src/test/resources/local-csv-to-print.json

```diff
@@ -0,0 +1,36 @@
+{
+  "job": {
+    "common": {
+      "job_id": -255,
+      "instance_id": -2035,
+      "job_name": "bitsail_local_csv_to_print_test",
+      "user_name": "test"
+    },
+    "reader": {
+      "class": "com.bytedance.bitsail.connector.local.csv.source.LocalCsvSource",
+      "file_path": "src/test/resources/test.csv",

```

Comment:
Do we really need so many data to test?
I mean, our test is just to verify whether the function is correct. In fact, dozens of data are enough, and 200000 data are not needed.

Reply:
you are right, and I decreased csv data volume to 12 rows